profiler/setup.py

```diff
@@ -4,5 +4,5 @@
     name="soma-profiler",
     version="1.0",
     packages=find_packages(),
-    requires=["gitpython", "psutil"],
+    requires=["gitpython", "comacore", "psutil", "tiledbsoma", "cellxgene_census"],
```

Comment:
`somacore`. Though that shouldn't be needed since `tiledbsoma` is required (transitive dependency)